Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.yml

```diff
@@ -1789,19 +1786,28 @@ script:
       required: false
       secret: false
     - default: false
-      description: The IP address or an array of IP addresses for
-        which to search. For example 1.1.1.1,2.2.2.2. Used instead of the source/destination IP address.
+      description: The IP address or an array of IP addresses for which to search.
+        For example 1.1.1.1,2.2.2.2. Used instead of the source/destination IP address.
       isArray: false
       name: ip
       required: false
       secret: false
     - default: false
-      description: The port or array of ports to
-        search. Used instead of the source/destination port.
+      description: The port or array of ports to search. Used instead of the source/destination
+        port.
       isArray: false
       name: port
       required: false
       secret: false
+    - default: false
```

Comment:
this param is in the code already?

Reply:
![Screen Shot 2020-08-05 at 14 53 30](https://user-images.githubusercontent.com/54398957/89409880-72559b80-d72b-11ea-88d3-1e7e1f6a4da6.png)

yes, only missing on the yml file.